index.js

```diff
@@ -16,7 +16,8 @@ const fsUtil = require('./src/util/fsUtil');
 const logger = require('./src/util/logger');
 const Site = require('./src/Site');
 
-const ACCEPTED_COMMANDS = ['version', 'init', 'build', 'serve', 'deploy'];
+const ACCEPTED_COMMANDS = ['init', 'build', 'serve', 'deploy'];
```

Comment:
Removed `version` from this list because it's technically not a command / `markbind version` is a no-op